orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Orchestration.groovy

```diff
@@ -20,7 +20,5 @@ import groovy.transform.CompileStatic
 
 @CompileStatic
 class Orchestration extends Execution<Orchestration> {
-  static final int CURRENT_VERSION = 2
```

Comment:
It might not hurt to keep the `CURRENT_VERSION` around for the next time a migration is necessary? Any harm in that?